app/helpers/application_helper.rb

```diff
@@ -290,8 +290,6 @@ def admin_config_description(content)
   end
 
   def role_display_name(role)
-    # TODO: [@jacobherrington] After all Forems have successfully deployed and the banned role
```

Comment:
This role has been removed quite a while ago, the helper can now be simplified again.